Node.gitignore

```diff
@@ -5,6 +5,9 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# Diagnostic reports
```

Comment:
Is it worth including a link to the documentation about this, given it's an experimental feature?

Reply:
Done.